src/content/overview/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Introduction to Giant Swarm
+description: Giant Swarm give platform engineering teams the ability to build cloud-native developer platforms and operate them without much hassle.
+weight: 10
+menu:
+  main:
+    parent: overview
+user_questions:
+  - What is a cloud-native developer platform?
+  - How does Giant Swarm help engineering teams?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+last_review_date: 2024-03-12
+---
+
+Giant Swarm's mission is to empower our customers to build their own cloud-native [developer platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#what-is-a-platform). As engineering teams grow and their environments expand, they often face exponentially growing complexity within the cloud-native ecosystem. This can be overwhelming. We believe that in most cases, having a [unified platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#why-platforms) plays a crucial role in keeping or gaining control over the entropic nature of such complex environments, while also significantly enhancing the users experience in using them.
```

Comment:
```suggestion
Giant Swarm's mission is to empower our customers to build their own cloud-native [developer platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#what-is-a-platform). When development organizations and departments grow and their environments expand, they often face exponentially growing complexity to manage the lifecycle and operations of their application and services. This easily gets overwhelming. We believe that having a flexible [unified platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#why-platforms) plays a crucial role in growing your development organization efficiently. The platform not only helps to take the toil out of the development teams to have less cognitive load, but also enables capabilities to manage security and governance more centrally.
```

Reply:
Very nice. Just a few thoughts which I wasn't able to put into good sentences:

- Should we mention shortly _what_ is overwhelming? Something like "creating and maintaining it all yourself while it's not your core business".
- Link to what ["toil"](https://sre.google/sre-book/eliminating-toil/) is? I don't think the term is widely known in non English natives at all, unless they have read the Google SRE book.

And suggestions on top of Timo's:

- `flexible, unified cloud-native developer platform` to bring the CNCF term in (and a comma)
- `s/to manage/for managing/`